docs/.env.template

```diff
@@ -5,3 +5,6 @@
 # Flag that determines whether or not repo-local API documentation should be generated and included in the site build.
 # Only used in local development.
 LOCAL_API_DOCS=<boolean>
+
+# This is the instrumentation key for the Application Insights config.
```

Comment:
Nit: probably worth either adding a bit more detail here so devs know how/why to use this, or point the reader to the README for more details.